### ISSUE TITLE
Accessibility Policy

### ISSUE THREAD

**Issue by conail** [dippam/client#6]
The accessibility policy from the old site is missing and needs an appropriate replacement.

**Comment by conail** [comment]
Examples:
- [gov.uk](https://www.gov.uk/help/accessibility): describes tool integration and where to get more help.
- [Economist](http://www.economist.com/help/accessibilitypolicy): describes design steps.
